src/main/scala/io/iohk/ethereum/faucet/FaucetApi.scala

```diff
@@ -40,7 +40,7 @@ class FaucetApi(rpcClient: RpcClient, keyStore: KeyStore, config: FaucetConfig,
 
   private def handleRequest(clientAddr: RemoteAddress, targetAddress: String) = {
     val timeMillis = clock.instant().toEpochMilli
-    val latestRequestTimestamp = latestRequestTimestamps.getOrElse(clientAddr, 0L)
+    val latestRequestTimestamp = latestRequestTimestamps.getOrDefault(clientAddr, 0L)
```

Comment:
shouldn't cleaning be implemented now as mentioned in twitter util collection changelog? I see a memory leak here now since LRUMap's purpose is to clean least used entries when full.